From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Bradley <daniel@pulumi.com>
Date: Wed, 22 May 2024 13:25:24 +0100
Subject: [PATCH] Avoid ordering container registry georeplications

This looks like it might have been removed in error due to a bad merge at some point.

diff --git a/internal/services/containers/container_registry_resource.go b/internal/services/containers/container_registry_resource.go
index 1c7aa455bbd9..1b739ab47bb9 100644
--- a/internal/services/containers/container_registry_resource.go
+++ b/internal/services/containers/container_registry_resource.go
@@ -10,7 +10,6 @@ import (
 	"log"
 	"net/http"
 	"slices"
-	"sort"
 	"strings"
 	"time"
 
@@ -870,11 +869,6 @@ func resourceContainerRegistryRead(d *pluginsdk.ResourceData, meta any) error {
 		}
 	}
 
-	// The order of the georeplications returned from the list API is not consistent. We simply order it alphabetically to be consistent.
-	sort.Slice(geoReplications, func(i, j int) bool {
-		return geoReplications[i].(map[string]any)["location"].(string) < geoReplications[j].(map[string]any)["location"].(string)
-	})
-
 	d.Set("georeplications", geoReplications)
 
 	return nil
